### ISSUE TITLE
Game crashed whilst tesselating block in world

### ISSUE THREAD

**Issue by bales75** [asim0v/GamealitionPack#8]
Attempting to login but my client keeps crashing with the message:

> The game crashed whilst tesselating block in world
> Error: java.lang.NullPointerException: Tesselating block in world"

Still trying to troubleshoot to see if it's client side, but here's the crash report.

https://gist.github.com/bales75/980350a1be84942d28c5e71286c634c8

It first happened when I teleported, via rftools teleporter, back to my main base.  When I arrived I loaded in just fine, but as soon as I started to turn around (I was facing towards the wall in my base), the game crashed.  I'm able to move around when I load in the game, but as soon as I look towards the general direction of my XNet connectors, that's when it crashes (assuming that's what's causing the crashes from the crash reports).

**Comment by bales75** [comment]
I must have moved too far, because now I'm not able to load at all.  Immediate crash to desktop.  My avatar is somewhere around 1793, 1043.  A system reboot did not work, either.

**Comment by bales75** [comment]
https://github.com/McJty/XNet/issues/168

**Comment by asim0v** [comment]
I have just updated the pack to include the xnet update. Waiting for the file to get approved on CurseForge before updating the server.

**Comment by asim0v** [comment]
After updating to the most current XNet and trying it again, I still crash at your place. Went looking into my crash report, and it's actually different than yours. https://gist.github.com/asim0v/cdea967e7895903330ce87f9acb42188

My crash log is pointing to an interaction between Immersive Engineering belljars (garden cloche) and Extra Utilities transfer nodes/pipes.

EDIT: https://github.com/BluSunrize/ImmersiveEngineering/issues/2560

**Comment by asim0v** [comment]
Still waiting on an update from Immersive Engineering, but I was able to MCEdit the Extra Utilities pipes and transfer node out that were causing the crash. Players can once again enter the area without crashing.

Going to close this for now since the specific case has been fixed. The issue, however, could pop up again. If it does, I'll have to manually edit the region files to remove the offending blocks again.
